logging_output_scripts/latex_tabulars.py

```diff
@@ -0,0 +1,148 @@
+    for i in dirs:
+        directory = dirs[i]
+        df = pd.read_csv(f"../{directory}/Results.csv")
+        comp_list = []
+        for j in comp_column:
+            c = comp_column[j]
+            data_res = load_problem_columns(df, c)
+            comp_list.append((comp_column_short[j], data_res[0], data_res[1], data_res[2], data_res[3], data_res[4]))
+
+        res = tabulate(comp_list, tablefmt="latex", headers=datasets_short.values())
+        with open(f"Complexity/Complexity-{directory}.txt", "w") as file:
+            file.write(res)
+
+
+# GENOMES
+def write_genomes():
+    for i in dirs:
+        directory = dirs[i]
+        for j in datasets:
+            dataset = datasets[j]
+
+            with open(f'Genomes/{directory}_{dataset}.json', 'r') as f:
+                genomes = json.load(f)
+
+            genomes = list(genomes.values())
+
+            genome_list = []
+            for x in range(len(genomes)):
+                temp = genomes[i]
+                genome_list.append((str(x), temp))
+
+            headers = ['Iteration', 'Genome']
+            res = tabulate(genome_list, tablefmt="latex", headers=headers)
+            with open(f"Genomes/{directory}-{dataset}.txt", "w") as file:
+                file.write(res)
+
+
+# MSE TABLES (Creates individual tables to be combined into a larger tabular)
+def write_mse():
+    row = []
+    for v in datasets.values():
+        temp = []
+        for i in dirs:
+            directory = dirs[i]
+            df = pd.read_csv(f"../{directory}/Results.csv")
+            res = df[df['Problem'].str.contains(v)]
+            temp.append(((round(float(res['MEAN_MSE']), 4)), round(float(res['STD_MSE']), 4)))
+        row.append(temp)
+
+    headers = ['MSE', 'STD']
+    table_1 = tabulate(row[0], tablefmt="latex", headers=headers)
+    table_2 = tabulate(row[1], tablefmt="latex", headers=headers)
+    table_3 = tabulate(row[2], tablefmt="latex", headers=headers)
+    table_4 = tabulate(row[3], tablefmt="latex", headers=headers)
+    table_5 = tabulate(row[4], tablefmt="latex", headers=headers)
+
+    with open(f"MSE/MSE.txt", "w") as file:
+        file.write(table_1 + "\n\n" + table_2 + "\n\n" + table_3 + "\n\n" + table_4 + "\n\n" + table_5)
+
+
+# CONVERGENCE TABLES
+def write_convergence():
+    for i in dirs:
+        directory = dirs[i]
+        df = pd.read_csv(f"../{directory}/Results.csv")
+        comp_list = []
+        for j in thresh_column:
+            c = thresh_column[j]
+            data_res = load_problem_columns(df, c)
+            if j % 3 == 0:
+                threshold = int(j / 3)
+                comp_list.append(
+                    (f'Threshold = {threshold}', thresh_column_short[i], data_res[0], data_res[1],
+                     data_res[2], data_res[3], data_res[4]))
+            else:
+                comp_list.append((" ", thresh_column_short[i], data_res[0],
+                                  data_res[1], data_res[2], data_res[3], data_res[4]))
```

Comment:
Why do we need this `if/else`? 
If we need it, you can write it like this to make it more readable:
```
if j % 3 == 0:
    threshold = int(j / 3)
    threshold_string = f'Threshold = {threshold}'

comp_list.append((threshold_string, thresh_column_short[i], data_res[0],
                                  data_res[1], data_res[2], data_res[3], data_res[4]))
```